src/nerve_tools/configure_nerve.py

```diff
@@ -204,43 +223,39 @@ def generate_configuration(services, heartbeat_path, hacheck_port, weight, zk_to
         for depth, loc in enumerate(available_locations)
     }
 
-    for (service_name, service_info) in services:
-        port = service_info.get('port')
-        if port is None:
-            continue
-
-        mode = service_info.get('mode', 'http')
-        healthcheck_timeout_s = service_info.get('healthcheck_timeout_s', 1.0)
-        healthcheck_port = service_info.get('healthcheck_port', port)
-
-        # hacheck will simply ignore the healthcheck_uri for TCP mode checks
-        healthcheck_uri = service_info.get('healthcheck_uri', '/status')
-        healthcheck_mode = service_info.get('healthcheck_mode', mode)
-        hacheck_uri = '/%s/%s/%s/%s' % (
-            healthcheck_mode, service_name, healthcheck_port, healthcheck_uri.lstrip('/'))
-        advertise = service_info.get('advertise', ['region'])
-        extra_advertise = service_info.get('extra_advertise', [])
-        extra_healthcheck_headers = service_info.get('extra_healthcheck_headers', {})
-
+    def update_subconfiguration_for_here(
```

Comment:
This is kinda a funny name for a function, but I can't think of any super better one